composer.json

```diff
@@ -392,8 +392,7 @@
                 "Automated Drupal 10 compatibility fixes - https://www.drupal.org/project/jira_rest/issues/3288088#comment-14567776": "https://www.drupal.org/files/issues/2022-06-16/jira_rest.4.x-dev.rector.patch"
             },
             "drupal/webform": {
-                "Exporting webform submission as batch does not allowed for extended field due to static method - https://www.drupal.org/project/webform/issues/3348336#comment-15686345": "https://www.drupal.org/files/issues/2024-07-24/exporting-webform-submission-static-batch-process-3348336-8.patch",
-                "Fix incompatibility with devel ^5.2 - https://www.drupal.org/project/webform/issues/3524877#comment-16111172": "https://www.drupal.org/files/issues/2025-05-16/webform-6.3.0-beta2-3524877.patch"
+                "Exporting webform submission as batch does not allowed for extended field due to static method - https://www.drupal.org/project/webform/issues/3348336#comment-16202432": "https://www.drupal.org/files/issues/2025-07-23/exporting-webform-submission-static-batch-process-3348336-9.patch"
```

Comment:
Regarding this patch, as I see it, it will never get merged in as it is specific to our field. Can this be extended via a hook instead?